lib/Resources/config/internal.yml

```diff
@@ -36,5 +46,24 @@ services:
         class: Netgen\EzPlatformSiteApi\Core\Site\Site
         lazy: true
         arguments:
+            - '@netgen.ezplatform_site.filter_service'
             - '@netgen.ezplatform_site.find_service'
             - '@netgen.ezplatform_site.load_service'
+
+    netgen.ezplatform_site.inner_repository:
+        class: '%ezpublish.api.inner_repository.class%'
+        factory:
+            - '@ezpublish.api.repository.factory'
+            - buildRepository
+        arguments:
+            - '@ezpublish.api.persistence_handler'
+            - '@ezpublish.spi.search.legacy'
+        lazy: true
+        public: false
+
+    netgen.ezplatform_site.repository.filter_service:
```

Comment:
Added in https://github.com/netgen/ezplatform-site-api/pull/31/commits/6f9f085b2a203a2ef15fd2a5f294d1134b794eb8